sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
We [aren't]( https://github.com/gratipay/gratipay.com/blob/61fbb2a4baa00d2da2d8cc457a890b4af71abd4d/tests/py/test_billing_payday.py#L119) otherwise resetting to zero after a card fails. Maybe we should make an effort to avoid doing that in the repair step here?